Modules/xASL_module_ASL.m

```diff
@@ -190,6 +195,54 @@
 	x.Q.NumberEchoTimes = 1;
 end
 
+%% D6. Automatic session merging
+
+% SessionMergingList == total list of sessions that need concatenating (can be multiple lists for multiple concatenations)
+% Read the list of sessions to merge, defaulting to empty
+if ~isfield(x.modules.asl, 'SessionMergingList')
+	x.modules.asl.SessionMergingList = {};
+else
+	% The parameter should be a list of lists, but if a single list is provided, it converts it
+	if ~isempty(x.modules.asl.SessionMergingList) && iscell(x.modules.asl.SessionMergingList) && ~iscell(x.modules.asl.SessionMergingList{1})
+		x.modules.asl.SessionMergingList = {x.modules.asl.SessionMergingList};
+    end
+    nLists = numel(x.modules.asl.SessionMergingList);
```

Comment:
We still have to check if the list is not empty... Because now we only know the variable exist but it might be empty

Reply:
Added nLists = 0 as dummy in 06081dd879b63b008be821d6baabc965cda9e5e5